src/ngraph/frontend/onnx_import/utils/reshape.cpp

```diff
@@ -266,6 +266,22 @@ namespace ngraph
                 return split(node, length_parts, axis_to_split);
             }
 
+            std::shared_ptr<ngraph::Node>
+                interpret_as_scalar(const std::shared_ptr<ngraph::Node>& node)
+            {
+                Shape node_shape = node->get_shape();
+                NGRAPH_CHECK((shape_size(node_shape) == 1),
+                             "Scalar value can't be derived from a node with ",
```

Comment:
"(...) node with shape:" Will be more clear error message.

Reply:
`node_shape` resolves to `Shape{1,2}`, so `node with shape:` will generate `node with shape: Shape{1,2}`